modules/sor/sorV2/beetsHelpers.ts

```diff
@@ -1,6 +1,7 @@
-import { BatchSwapStep, NATIVE_ADDRESS, SingleSwap, SwapKind, ZERO_ADDRESS } from '@balancer/sdk';
 import { GqlPoolMinimal, GqlSorSwapRoute, GqlSorSwapRouteHop } from '../../../schema';
 import { formatFixed } from '@ethersproject/bignumber';
+import { BatchSwapStep, SingleSwap, SwapKind } from './sor-port/types';
+import { NATIVE_ADDRESS, ZERO_ADDRESS } from './sor-port/constants';
```

Comment:
I'm thinking if we shouldn't keep the SOR self-contained and import only from one module exposing the interface internally to the API, ideally without touching any internal types.